tests/integration/docusaurus/expectations/expectation_management.py

```diff
@@ -0,0 +1,56 @@
+# setup
+import great_expectations as gx
+from great_expectations.core.expectation_suite import ExpectationSuite
+from great_expectations.expectations.core import (
+    ExpectColumnValuesToBeInSet,
+    ExpectColumnValuesToNotBeNull,
+)
+
+context = gx.get_context()
+suite_name = "test-suite"
+column_a = "a"
+column_b = "b"
+column_c = "c"
+
+# create a suite
+suite = ExpectationSuite(name=suite_name)
+suite = context.suites.add(suite=suite)
+
+
+# add expectations
+expectation_a = ExpectColumnValuesToBeInSet(
+    column=column_a, value_set=["foo", "bar", "baz"]
+)
+suite.add_expectation(expectation=expectation_a)
+expectation_b = ExpectColumnValuesToNotBeNull(column=column_a)
+suite.add_expectation(expectation=expectation_b)
+
+# find and update a single expectation
+expectation_type = ExpectColumnValuesToNotBeNull
```

Comment:
Non-blocking: same basic thought as above - this assignment feels like just an empty line that isn't doing anything to improve readability. `isinstance(expectation, ExpectColumnValuesToNotBeNull)` feels totally readable